src/inference/__tests__/embed.test.ts

```diff
@@ -0,0 +1,51 @@
+import { Inference } from '../inference';
+import type { PineconeConfiguration } from '../../data';
+import { inferenceOperationsBuilder } from '../inferenceOperationsBuilder';
+
+let inference: Inference;
+
+beforeAll(() => {
+  const config: PineconeConfiguration = { apiKey: 'test-api-key' };
+  const infApi = inferenceOperationsBuilder(config);
+  inference = new Inference(infApi);
+});
+
+describe('Inference Class: _formatInputs', () => {
+  test('Should format inputs correctly', () => {
+    const inputs = ['input1', 'input2'];
+    const expected = [{ text: 'input1' }, { text: 'input2' }];
+    const result = inference._formatInputs(inputs);
+    expect(result).toEqual(expected);
+  });
+});
+
+describe('Inference Class: _formatParams', () => {
+  test('Should format params correctly', () => {
+    const params = { inputType: 'text', truncate: 'END' };
+    const expected = { inputType: 'text', truncate: 'END' };
```

Comment:
Do we need to test if we're not expecting this to change anything? Maybe I'm unclear on the test.

Reply:
weird... this must be a holdover from something... I'll remove method (it does nothing currently) + this test. thanks!